integritybackend/asset_helper.py

```diff
@@ -67,21 +57,7 @@ def __init__(self, organization_id):
             config.INTERNAL_ASSET_STORE, organization_id
         )
         self.shared_prefix = os.path.join(config.SHARED_FILE_SYSTEM, organization_id)
-
-        # Internal directories
-        self.dir_internal_assets = os.path.join(self.internal_prefix, "assets")
-        self.dir_internal_claims = os.path.join(self.internal_prefix, "claims")
         self.dir_internal_tmp = os.path.join(self.internal_prefix, "tmp")
-        self.dir_internal_create = os.path.join(self.internal_prefix, "create")
-        self.dir_internal_create_proofmode = os.path.join(
-            self.internal_prefix, "create-proofmode"
-        )
-
-        # Legacy shared output directories (not collection-specific)
```

Comment:
I got rid of all legacy. Legacy Starling Capture is completely broke now but will be fixed in future PR.